packages/@aws-cdk/aws-ec2/lib/vpc-endpoint.ts

```diff
@@ -264,34 +264,56 @@ export class InterfaceVpcEndpointAwsService implements IInterfaceVpcEndpointServ
   public static readonly ACCOUNT_MANAGEMENT = new InterfaceVpcEndpointAwsService('account');
   public static readonly APP_MESH = new InterfaceVpcEndpointAwsService('appmesh-envoy-management');
   public static readonly APP_RUNNER = new InterfaceVpcEndpointAwsService('apprunner');
+  public static readonly APP_RUNNER_REQUESTS = new InterfaceVpcEndpointAwsService('apprunner.requests');
   public static readonly APPLICATION_MIGRATION_SERVICE = new InterfaceVpcEndpointAwsService('mgn');
+  public static readonly APPSTREAM = new InterfaceVpcEndpointAwsService('appstream.api');
```

Comment:
```suggestion
  public static readonly APPSTREAM_API = new InterfaceVpcEndpointAwsService('appstream.api');
```
I'm wondering if this should be consistent with what the endpoint actually is.